rcgen/src/certificate.rs

```diff
@@ -467,20 +468,18 @@ impl CertificateParams {
 		});
 	}
 
-	/// Generate and serialize a certificate signing request (CSR) in binary DER format.
+	/// Generate and serialize a certificate signing request (CSR).
 	///
 	/// The constructed CSR will contain attributes based on the certificate parameters,
 	/// and include the subject public key information from `subject_key`. Additionally,
 	/// the CSR will be self-signed using the subject key.
 	///
-	/// **Important**: since this function generates a new CSR for each invocation you
-	/// should not call `serialize_request_der` and then `serialize_request_pem`. This will
-	/// result in two different CSRs. Instead call only `serialize_request_pem` and base64
-	/// decode the inner content to get the DER encoded CSR using a library like `pem`.
-	pub fn serialize_request_der(
+	/// Note that subsequent invocations of `serialize_request()` will not produce exactly
+	/// the same output.
```

Comment:
editorial nit: I think this will read more naturally as "will not produce the exact same output" instead of "will not produce exactly the same output"